### DISCUSSION TITLE
Lost password

### DISCUSSION THREAD

**Discussion by RodHub** [akeeba/panopticon#668] [General]
I installed Panopticon, all worked well. Then I accidentally deleted my credentials from the browser, the next day on log-in. Now there is no password reset functionality, so I changed the password in the database, this also didn't work. I tried inserting a hashed password, also no luck.

Is starting over the only solution at this point?

**Comment by nikosdion** [comment]
Starting over is **NOT** a valid solution to such a trivial problem.

Panopticon stores the passwords in the `#__users` table, in the `password` column, hashed with bCrypt. Put `$2y$12$u76vDKZv/7sdqF.5I7n7gOSftWYf2cO3mmtJg8bwWbQfDiqAEt/Qi` in there and you can log in with the password `password`. Then, immediately change it back to something secure.

Note: the same password hash is used by Joomla! and WordPress. If you lose your password and there's no way to recover it, do the same thing. Put `$2y$12$u76vDKZv/7sdqF.5I7n7gOSftWYf2cO3mmtJg8bwWbQfDiqAEt/Qi` into the `password` field of the respective `#__users` table and you can log in with `password`.

**Comment by RodHub** [comment]
Thanks, I actually tried copying a password from one of my Joomla user accounts, that didn't work. But your solution did work 👍

**Reply by nikosdion** [reply]
That's weird. Joomla! has been using bCrypt since the early 3.x releases, possibly 2.5 (I would have to check). If you were using an older hashing algo, such as MD5 or salted MD5, it would upgrade it upon first login, meaning your database should only have bCrypt-hashed passwords for all active users.
